go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -780,15 +776,14 @@ var defaultConfig = TabletConfig{
 	// memory copies.  so with the encoding overhead, this seems to work
 	// great (the overhead makes the final packets on the wire about twice
 	// bigger than this).
-	StreamBufferSize:                        32 * 1024,
-	QueryCacheSize:                          int(cache.DefaultConfig.MaxEntries),
-	QueryCacheMemory:                        cache.DefaultConfig.MaxMemoryUsage,
-	QueryCacheLFU:                           cache.DefaultConfig.LFU,
-	SchemaReloadIntervalSeconds:             flagutil.NewDeprecatedFloat64Seconds("queryserver-config-schema-reload-time", 30*time.Minute),
-	SignalSchemaChangeReloadIntervalSeconds: flagutil.NewDeprecatedFloat64Seconds("queryserver-config-schema-change-signal-interval", 5*time.Second),
-	MessagePostponeParallelism:              4,
-	DeprecatedCacheResultFields:             true,
-	SignalWhenSchemaChange:                  true,
+	StreamBufferSize:            32 * 1024,
+	QueryCacheSize:              int(cache.DefaultConfig.MaxEntries),
+	QueryCacheMemory:            cache.DefaultConfig.MaxMemoryUsage,
+	QueryCacheLFU:               cache.DefaultConfig.LFU,
+	SchemaReloadIntervalSeconds: flagutil.NewDeprecatedFloat64Seconds("queryserver-config-schema-reload-time", 30*time.Minute),
```

Comment:
I don't think we should change the default. The entire benefit of this schema tracker is that it runs automatically when a DDL happens through Vtgates. Because of this, we don't have to poll MySQL very aggressively. I think that the use-case where people execute DDLs directly in MySQL isn't something for which we should change the default. I think if someone is doing that, then it falls under extraordinary circumstances, they should change the flag value. The default should be a less aggressive poll time like we have now.